ironfish-cli/src/commands/peers/index.ts

```diff
@@ -47,6 +47,11 @@ export class ListCommand extends IronfishCommand {
       default: false,
       description: 'Display node names',
     }),
+    direction: Flags.boolean({
```

Comment:
I think we have an extended flag already we could just hide it behind that? Not sure it needs its own flag we might be getting into too many custom flags

Reply:
Ah good call out, I didn't know that's how the extended field works. Yeah totally agree, too many flags. I made the change.